portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,57 @@
 // Copyright 2019 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed under the Apache License, Version 2.0 (the 'License');
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
 //     https://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
+// distributed under the License is distributed on an 'AS IS' BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random activity card to the page. These are descriptions and images of random quarantine activities.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Gone Now by Bleachers', 'Malibu Nights by LANY', 'Melodrama by Lorde','After Hours by The Weeknd', 'American Teen by Khalid'];
+function addRandomActivity() {
```

Comment:
just a nit: can you give it a better name? Maybe just addActivity? It is not random anymore now :)